src/Symfony/Component/AssetMapper/Tests/Compiler/JavaScriptImportPathCompilerTest.php

```diff
@@ -34,6 +34,9 @@ public function testCompile(string $sourceLogicalName, string $input, array $exp
             $actualDependencies[$dependency->asset->logicalPath] = $dependency->isLazy;
         }
         $this->assertEquals($expectedDependencies, $actualDependencies);
+        if (\count($expectedDependencies) > 0) {
```

Comment:
we never "count" on array: 
```suggestion
        if ($expectedDependencies) {
```